teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -134,50 +131,44 @@ private static class ConfigFileCommand {
       scope = ScopeType.INHERIT)
   private File configFile;
 
-  @Mixin(name = "Network")
-  private Eth2NetworkOptions eth2NetworkOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Network%n")
+  private Eth2NetworkOptions eth2NetworkOptions = new Eth2NetworkOptions();
 
-  @Mixin(name = "P2P")
-  private P2POptions p2POptions;
+  @CommandLine.ArgGroup(validate = false, heading = "P2P%n")
+  private P2POptions p2POptions = new P2POptions();
 
-  @Mixin(name = "Interop")
-  private InteropOptions interopOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator%n")
+  private ValidatorOptions validatorOptions = new ValidatorOptions();
 
-  @Mixin(name = "Validator")
-  private ValidatorOptions validatorOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Execution Layer%n")
+  private ExecutionLayerOptions executionLayerOptions = new ExecutionLayerOptions();
 
-  @Mixin(name = "Deposit")
-  private DepositOptions depositOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Beacon Node Data%n")
+  private BeaconNodeDataOptions dataOptions = new BeaconNodeDataOptions();
 
-  @Mixin(name = "Execution Layer")
-  private ExecutionLayerOptions executionLayerOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Data Storage%n")
+  private DataStorageOptions dataStorageOptions = new DataStorageOptions();
```

Comment:
I wasn't able to merge DataStorageOptions into BeaconNodeDataOptions, and I'm a bit unfamiliar with how to resolve this part. BeaconNodeDataOptions is implemented differently in that its TekuConfiguration builder is implemented in the abstract class DataOptions. Trying to extend `public void configure(TekuConfiguration.Builder builder)` in BeaconNodeDataOptions causes the `:autocomplete` Gradle step to break during build.

Reply:
hmm, interesting. That sounds vaguely familiar.  I'll have a play around with it tomorrow and see what I can work out.

Reply:
So it turns out that everywhere we use `DataStorageOptions` we also use `BeaconNodeDataOptions` which is why autocomplete was breaking - the same option was getting included twice.  So I've merged `DataStorageOptions` into `BeaconNodeDataOptions`.